components/stacks-network/src/orchestrator.rs

```diff
@@ -1035,65 +1043,89 @@ events_keys = ["*"]
 
         stacks_conf.push_str(&format!(
             r#"
-[burnchain]
-chain = "bitcoin"
-mode = "krypton"
-poll_time_secs = 1
-timeout = 30
-peer_host = "host.docker.internal"
-rpc_ssl = false
-wallet_name = "devnet"
-username = "{bitcoin_node_username}"
-password = "{bitcoin_node_password}"
-rpc_port = {orchestrator_ingestion_port}
-peer_port = {bitcoin_node_p2p_port}
-"#,
+        [burnchain]
+        chain = "bitcoin"
+        mode = "{burnchain_mode}"
+        magic_bytes = "T3"
+        pox_prepare_length = 5
+        pox_reward_length = 10
+        burn_fee_cap = 20_000
+        poll_time_secs = 1
+        timeout = 30
+        peer_host = "host.docker.internal"
+        rpc_ssl = false
+        wallet_name = "{miner_wallet_name}"
+        username = "{bitcoin_node_username}"
+        password = "{bitcoin_node_password}"
+        rpc_port = {orchestrator_ingestion_port}
+        peer_port = {bitcoin_node_p2p_port}
+        "#,
```

Comment:
Can the indentation here match what it was before (left-aligned)?

This template string will be put in the file exactly as-is, so we'll have random blocks indented.